src/lightning/pytorch/callbacks/progress/tqdm_progress.py

```diff
@@ -190,7 +192,6 @@ def init_train_tqdm(self) -> Tqdm:
         """Override this to customize the tqdm bar for training."""
         bar = Tqdm(
             desc=self.train_description,
-            initial=self.train_batch_idx,
```

Comment:
What will we do about this? Don't you need to add an update to the on_train_start hooks etc?

Reply:
I don't think this is necessary:
- `on_train_epoch_start` already sets initial=0.
- When any batch runs, the `n` will be updated
- The other bars did not set it.

Reply:
See comment above.